docs/experiments/focus-metavariable.md

```diff
@@ -0,0 +1,27 @@
+---
+slug: focus-metavariable
+append_help_link: true
+description: "focus-metavariable allows Semgrep users to narrow down the code matched by a rule, to focus on what it really matters."
+---
+
+# `focus-metavariable`
+
+The experimental `focus-metavariable` operator puts the focus, or "zooms in", on the code region matched by a metavariable. For example, if we wanted to find all functions arguments annotated with the type `bad` we could write this pattern:
+
+```yaml
+pattern: |
+  def $FUNC(..., $ARG : bad, ...):
+    ...
+```
+
+This works but it matches the entire function definition. Sometimes that is not desirable, for instance, if we are using [Semgrep App](https://semgrep.dev/login) and we have triaged a finding generated by this pattern, the same finding may show up again as new if we make any change to the definition of the function!
+
+Using `focus-metavariable` we can tell Semgrep that we are only interested in the code matched by a particular metavariable, in our case, `$ARG`:
```

Comment:
```suggestion
To specify that you are only interested in the code matched by a particular metavariable, in our case `$ARG`, use `focus-metavariable`.
```

Reply:
(Just explanation of the suggestion above.) First, describe the goal (to do/achieve this and that), utilize this.

This makes the reading a little bit easier and it is more engaging for the reader.